HealthcareApp/src/Healthcare.Domain/Constants/AppointmentStatusConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace Domain.Constants;
+
+public enum AppointmentStatusConstants
```

Comment:
Enum is not constants, remove this in name.
Add numbers for enum members explicitly, because reordering will break the application if some values already stored in db